### PR TITLE
Basic updates to pom

### PR DESCRIPTION
**Pull request by MatthewKhouzam**
Build a jar

Add info

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,10 +7,10 @@
   <groupId>org.eclipse.tracecompass</groupId>
   <artifactId>trace-event-logger</artifactId>
   <version>0.0.3-SNAPSHOT</version>
-
+  <packaging>jar</packaging>
   <name>trace-event-logger</name>
   <!-- the project's website -->
-  <url>http://www.tracecompass.org</url>
+  <url>https://eclipse.dev/tracecompass</url>
```
Comment (bhufmann) [inline pom.xml:13]: Should that be the URL of the repo?

Comment (MatthewKhouzam) [reply pom.xml:13]: I think that would be the <scm> tag. not sure.

Comment (bhufmann) [reply pom.xml:13]: Ok

**Comment by MatthewKhouzam** [comment]
Thanks!

**Pull request merged**